tests/phpunit/AutoReview/ProjectCode/ProjectCodeProvider.php

```diff
@@ -99,6 +117,21 @@ final class ProjectCodeProvider
         TestFrameworkTypes::class,
         MutationsCollectorVisitor::class,
         ParentConnectorVisitor::class,
+        ArrayItemRemoval::class,
```

Comment:
I think some of these no longer exist.

Reply:
I'd also really prefer we do not add to this list

Reply:
>I think some of these no longer exist.

Indeed

>I'd also really prefer we do not add to this list

It's mostly about classes which are effectively tested by `SchemaConfigurationFactory`, it's just that you don't always have a 1:1 class unit test mapping